include/picongpu/fields/currentDeposition/Strategy.def

```diff
@@ -147,6 +147,18 @@ namespace traits
     };
 #endif
 
+#if( ALPAKA_ACC_GPU_HIP_ENABLED == 1 )
+        template<
```

Comment:
This line is misaligned

Reply:
Not sure, perhaps we could add a short description of why HIP a different default strategy?

Reply:
Add a short discription